dataland-e2etests/build.gradle.kts

```diff
@@ -31,6 +31,7 @@ dependencies {
     implementation(libs.log4j.to.slf4j)
     implementation("org.springframework.boot:spring-boot-starter-web")
     testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation("org.awaitility:awaitility")

```

Comment:
here you rely on the version specified by Spring (since you don't provide a version explicitly). Hence the entry in settings.gradle not used, so it can be removed there.

Reply:
done